internal/testhelpers/httprecorder/check_tokens.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
You don't think that it's a little bit strange to have a `shell` script in a go package? :/

Reply:
It depends on the way you look at it. Having a script in a Go package, next to the files it is related to doesn't seem strange to me...

But tell me which other location you prefer.